### ISSUE TITLE
Support dbh-specific equations

### ISSUE THREAD

**Issue by maurolepore** [forestgeo/fgeo.biomass#26]

**Comment by gonzalezeb** [comment]
there are  23 equations in `equation` table with some type of missing dbh range value (NRA, NI, NA)..

You can replace all of them if you want, but will the csv file also be updated? Otherwise I will do it.

I will make sure to mention that not just in the code but in our methods.

**Comment by maurolepore** [comment]
> You can replace all of them if you want, but will the csv file also be updated? Otherwise I will do it.

No, the kind of fix I would do in code is downstream from the .csv files. In my opinion, the more you can do directly the better. Every new code statement increases the chance of bugs.
